oauthenticator/azuread.py

```diff
@@ -94,6 +111,16 @@ async def authenticate(self, handler, data=None):
         # results in a decoded JWT for the user data
         auth_state['user'] = decoded
 
+        roles = decoded.get("roles", [])
```

Comment:
Just for readability.

```suggestion
        roles = auth_state['user'].get("roles", [])
```

Reply:
Addressed in https://github.com/jupyterhub/oauthenticator/pull/446/commits/6dc41a6df1e479a7a7048b7ddc49630319da40ea